website/docs/r/cdn_frontdoor_firewall_policy.html.markdown

```diff
@@ -0,0 +1,265 @@
+
+    match_condition {
+      match_variable     = "RemoteAddr"
+      operator           = "IPMatch"
+      negation_condition = false
+      match_values       = ["192.168.1.0/24"]
+    }
+
+    match_condition {
+      match_variable     = "RequestHeader"
+      selector           = "UserAgent"
+      operator           = "Contains"
+      negation_condition = false
+      match_values       = ["windows"]
+      transforms         = ["Lowercase", "Trim"]
+    }
+  }
+
+  managed_rule {
+    type    = "DefaultRuleSet"
+    version = "1.0"
+
+    exclusion {
+      match_variable = "QueryStringArgNames"
+      operator       = "Equals"
+      selector       = "not_suspicious"
+    }
+
+    override {
+      rule_group_name = "PHP"
+
+      rule {
+        rule_id = "933100"
+        enabled = false
+        action  = "Block"
+      }
+    }
+
+    override {
+      rule_group_name = "SQLI"
+
+      exclusion {
+        match_variable = "QueryStringArgNames"
+        operator       = "Equals"
+        selector       = "really_not_suspicious"
+      }
+
+      rule {
+        rule_id = "942200"
+        action  = "Block"
+
+        exclusion {
+          match_variable = "QueryStringArgNames"
+          operator       = "Equals"
+          selector       = "innocent"
+        }
+      }
+    }
+  }
+
+  managed_rule {
+    type    = "Microsoft_BotManagerRuleSet"
+    version = "1.0"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the policy. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group. Changing this forces a new resource to be created.
+
+* `cdn_frontdoor_profile_id` - (Required) The resource ID of the Cdn Frontdoor Profile this firewall policy will belong. Changing this forces a new resource to be created.
+
+* `enabled` - (Optional) Is the policy a enabled state or disabled state. Defaults to `true`.
+
+* `mode` - (Optional) The firewall policy mode. Possible values are `Detection`, `Prevention` and defaults to `Prevention`.
+
+* `redirect_url` - (Optional) If action type is redirect, this field represents redirect URL for the client.
+
+* `custom_rule` - (Optional) One or more `custom_rule` blocks as defined below.
+
+* `custom_block_response_status_code` - (Optional) If a `custom_rule` block's action type is `block`, this is the response status code. Possible values are `200`, `403`, `405`, `406`, or `429`.
+
+* `custom_block_response_body` - (Optional) If a `custom_rule` block's action type is `block`, this is the response body. The body must be specified in base64 encoding.
+
+* `managed_rule` - (Optional) One or more `managed_rule` blocks as defined below.
+
+* `sku_name` - (Optional) The sku's pricing tier for this Cdn Frontdoor firewall policy. Possible values include`Standard_AzureFrontdoor` or `Premium_AzureFrontdoor`. Defaluts to `Standard_AzureFrontdoor`.
```

Comment:
```suggestion
* `sku_name` - (Optional) The sku's pricing tier for this Cdn Frontdoor firewall policy. Possible values include`Standard_AzureFrontDoor` or `Premium_AzureFrontDoor`. Defaluts to `Standard_AzureFrontDoor`.
```

Reply:
@sebader there's still a typo in there 'Defaluts' instead of 'Defaults'.

Reply:
Fixed.